coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 46.8,
+  "coverage_score": 18.2,
   "exclude_path": "",
```

Comment:
this is a little dissapointing because we add a lot of code and it seems fairly unexercised by unit tests.